config-template.json

```diff
@@ -14,27 +14,26 @@
         {
           "id": "98FF26D0-2E01-11E5-9184-FEFF819CDC9F",
           "name": "s3",
-          "description": "Amazon Simple Storage Service (S3)",
+          "description": "Amazon S3 provides developers with secure, durable, highly-scalable object storage",
           "bindable": true,
           "tags": [
             "AWS",
              "S3",
              "object-storage"
           ],
           "metadata": {
-            "displayName": "Amazon Simple Storage Service (S3)",
+            "displayName": "Amazon s3 storage",
```

Comment:
(We should use S3 instead of s3 as our consistent style, since Amazon calls it S3.)